tests/tests.xml

```diff
@@ -15,7 +15,7 @@
         <step expected_result="0">/opt/tests/ssu/runtest.sh ut_ssu</step>
       </case>
       <case name="ut_ssucli" type="Functional" description="ssu command line utility test" timeout="1000" subfeature="">
-        <step expected_result="0">/opt/tests/ssu/runtest.sh ut_ssucli</step>
+        <step expected_result="0">/usr/sbin/run-blts-root /opt/tests/ssu/runtest.sh ut_ssucli</step>
```

Comment:
This doesn't seem correct. The test suite should not require superuser rights - by design. What is the particular issue here when run under regular user account?

Reply:
The ssu segfaults.

Reply:
Per minimum this needs to make sure that potentially running dbus-org.nemo.ssu is stopped before starting test case. However, just by doing that is not enough.

I'm also open to hand this over to you.

Reply:
The test is not meant to use the system bus. It starts its own bus here https://github.com/sailfishos/ssu/blob/master/tests/ut_ssucli/ssuclitest.cpp#L36 . Could you please paste a backtrace?